Assignment/Assignment.Tests/SampleDataTests.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Assignment.Tests
+{
+    [TestClass]
+    public class SampleDataTests
+    {
+
+        SampleData data = new SampleData();
+
+        [TestMethod]
+        public void CsvRows_NotNull()
+        {
+            Assert.IsNotNull(data.CsvRows);
+        }
+
+        [TestMethod]
+        public void Csv_SkipFirstRow()
```

Comment:
```suggestion
        public void CsvRows_SkipFirstRow()
```
[UnitOfWork_StateUnderTest_ExpectedBehavior]